### DISCUSSION TITLE
"flask db init" causes error

### DISCUSSION THREAD

**Discussion by gurupetach** [miguelgrinberg/Flask-Migrate#416] [Q&A]
"flask db init "causes this error:     No handlers could be found for logger "flask_migrate".
What shoid I do? 
My alembic ini file is as below
# A generic, single database configuration.

[alembic]
# template used to generate migration files
# file_template = %%(rev)s_%%(slug)s

# set to 'true' to run the environment during
# the 'revision' command, regardless of autogenerate
# revision_environment = false

# Logging configuration
[loggers]
keys = root,sqlalchemy,alembic,flask_migrate

[handlers]
keys = console

[formatters]
keys = generic

[logger_root]
level = WARN
handlers = console
qualname =

[logger_sqlalchemy]
level = WARN
handlers =
qualname = sqlalchemy.engine

[logger_alembic]
level = INFO
handlers =
qualname = alembic

[logger_flask_migrate]
level = INFO
handlers =
qualname = flask_migrate

[handler_console]
class = StreamHandler
args = (sys.stderr,)
level = NOTSET
formatter = generic

[formatter_generic]
format = %(levelname)-5.5s [%(name)s] %(message)s
datefmt = %H:%M:%S

**Comment by miguelgrinberg** [comment]
Here is something I don't understand. The `flask db init` command creates the alembic.ini file. So how come you already have this file if the command isn't running?
